css/index.css

```diff
@@ -143,4 +221,174 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* About page */
+.about-header {
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed #1d1d1d;
+}
+
+.about-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.about-block {
+    width: 388px;
+    border-bottom: 1px dashed #1d1d1d;
+    margin-bottom: 20px;
+    padding: 10px; 
+}
+
+.about-block h2 {
+    padding-top: 10px;
+}
+
+.about-block button {
+    font-size: 16px;
+    margin: 15px 0px;
+    background: white;
+    border: 2px solid #1d1d1d;
+    border-radius: 8px;
+    padding: 8px 30px;
+}
+
+.about-footer {
+    padding-bottom: 20px;
+    margin: 20px 0px;
+}
+
+/* Box colors */
+
+#box1 {
+    background: teal;
+}
+
+#box2 {
+    background: gold;
+}
+
+#box3 {
+    background: cadetblue;
+}
+
+#box4 {
+    background: coral;
+}
+
+#box5 {
+    background: crimson;
+}
+
+#box6 {
+    background: forestgreen;
+}
+
+#box7 {
+    background: darkorchid;
+}
+
+#box8 {
+    background: hotpink;
+}
+
+#box9 {
+    background: indigo;
+}
+
+#box10 {
+    background: dodgerblue;
+}
+
+#box11 {
+    background: dodgerblue;
+}
+
+#box12 {
+    background: indigo;
+}
+
+#box13 {
+    background: hotpink;
+}
+
+#box14 {
+    background: darkorchid;
+}
+
+#box15 {
+    background: forestgreen;
+}
+
+#box16 {
+    background: crimson;
+}
+
+#box17 {
+    background: coral;
+}
+
+#box18 {
+    background: cadetblue;
+}
+
+#box19 {
+    background: gold;
+}
+
+#box20 {
+    background: teal;
+}
+
+/* Stretch Goals */
+/* entire container, keeps perspective */
+.flip-container {
+	perspective: 1000px;
+}
+
+/* flip the pane when hovered */
+.flip-container:hover .flipper, .flip-container:hover .flipper {
+    transform: rotateY(180deg);
+}
+
+.flip-container, .box-front, .box-back {
+    width: 100px;
+    height: 100px;
+    margin: 20px;
+    color: white;
+}
+
+/* flip speed goes here */
+.flipper {
+	transition: 0.6s;
+	transform-style: preserve-3d;
+
+	position: relative;
+}
+
+/* hide back of pane during swap */
+.box-front, .box-back {
```

Comment:
would like to hear your answer for what's the difference between `.box-front .box-back` and `.boxfront, box-back` (what does the comma do?)

Reply:
`.box-front .box-back` is only styling `.box-back` classes that are nested within `.box-front` classes.

`.box-front, .box-back` is styling both of the classes individually. If there were a bunch of elements with `color: goldenrod` then they could all be styled with `.class-1, .class-2, ..., .class-n` which would save a lot of time in the future if you want to change the color (or whatever other attribute it is).